### PR TITLE
Portfolio: setup and mobile first

### PR DESCRIPTION
**Pull request by MohammadYaser**
Set up a new repository and prepare it for development using best practices (e.g. linters).
Build the first 2 sections of the mobile website using the template you chose in the previous step.

### CODE REVIEW

**Comment by MohammadYaser** [comment]
style header section and headline section using flex box
 remove linter errors in css file

File: Index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+  <body>
+    <div class="mobile-container">
+      <nav class="topnav">
+        <a href="#home" class="active">My Logo</a>
+
+        <a class="icon">
+          <img src="images/btn.png" />
+        </a>
+      </nav>
```
Comment (mahbubul14) [inline Index.html]: - [ ] The container of the section should be wrapped in a `header` tag. Kindly work on it. :+1:

File: Index.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        <p class="text-content">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hesitate to contact me.
+        </p>
+        <img class="fa-icon" src="images/github.svg" />
+        <img class="fa-icon" src="images/Linkedin icon.svg" />
+        <img class="fa-icon" src="images/angel list.svg" />
+        <img class="fa-icon" src="images/twitter.svg" />
+        <img class="fa-icon" src="images/medium.svg" />
```
Comment (mahbubul14) [inline Index.html]: - [ ] The container of the section should be wrapped in a `section` tag. Please work on it :+1:

File: Index.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    <div class="mobile-container">
+      <nav class="topnav">
+        <a href="#home" class="active">My Logo</a>
+
+        <a class="icon">
+          <img src="images/btn.png" />
+        </a>
+      </nav>
+      <main>
+        <p class="headline">Hey There. I’m Yaser.</p>
```
Comment (mahbubul14) [inline Index.html]: - [ ] The main title (e.g., "I'm Marianne ...") should be in a `h1` tag. You should change `p` tag  to `h1` tag here.

File: Index.html (hunk 2 above)
Comment (mahbubul14) [inline Index.html]: - [ ] The list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Please do the change. :+1:

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
**Hi @MohammadYaser,**

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] In your `README` file, Kindly update the getting started section. This should be an example of how you may give instructions on setting up your project locally.

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

* N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by shahadat3669 — CHANGES_REQUESTED** [review]
Hi @MohammadYaser , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ No linter error.
✔️ Proper Github flow used.
✔️ Descriptive PR title & description

 ## Required Changes ♻️
Great work on updating the README file based on the previous reviewer's requests. It's important to carefully go through each requirement and address any inline requests made by the reviewer.

Please make sure to review the entire feedback provided by the previous reviewer and ensure that you have addressed all the requested changes. This includes following any specific instructions or suggestions provided in the inline comments.

By thoroughly reviewing and addressing all the previous reviewer's requests, you can ensure that your project meets the desired standards and requirements.

here is a[ link ](https://github.com/MohammadYaser/Portfolio/pull/1#discussion_r1214397896)for the requester of the previous reviewer.

image

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Index.html (hunk 1 above)
Comment (MohammadYaser) [reply Index.html:19]: you mean i should put all of this code inside a header tag? like this
<header>

      <nav class="topnav">
        <a href="#home" class="active">My Logo</a>

        <a class="icon">
          
        </a>
      </nav>
</header

File: Index.html [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+      <header class="topnav">
+        <nav>
+          <a href="#home" class="active">My Logo</a>
+          <a class="icon"><img src="images/btn.png" /></a>
+        </nav>
+      </header>
+      <section>
+        <main>
+          <h1 class="headline">Hey There. I’m Yaser.</h1>
+          <p class="title">I'm a Software Developer</p>
```
Comment (tufoinnkuo10) [inline Index.html]: - [optional] please, all the complete title should be in the h1 tag. Kindly adjust that.

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @MohammadYaser 
Your project is complete! Great job for making the changes 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- design followed
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**